inference-engine/tests/functional/plugin/cpu/test_utils/fusing_test_utils.hpp

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "cpu_test_utils.hpp"
+#include <shared_test_classes/single_layer/activation.hpp>
+
+namespace CPUTestUtils {
+
+struct postNode {
+    std::function<std::shared_ptr<ngraph::Node>(std::shared_ptr<ngraph::Node>, const ngraph::element::Type&, ngraph::ParameterVector&)> makeNode;
+    std::string name;
+};
+
+typedef std::tuple<
+        std::shared_ptr<ngraph::Function>,
+        std::vector<postNode>,
+        std::vector<std::string>> fusingSpecificParams;
+
+class CpuTestWithFusing : public CPUTestsBase {
+public:
+    static std::string getTestCaseName(fusingSpecificParams params);
+
+protected:
+    std::shared_ptr<ngraph::Node> modifyGraph(const ngraph::element::Type &ngPrc,
+                                                      ngraph::ParameterVector &params,
+                                                      const std::shared_ptr<ngraph::Node> &lastNode) const override;
+
+    void CheckPluginRelatedResults(InferenceEngine::ExecutableNetwork &execNet, std::string nodeType) const override;
+
+private:
+    std::shared_ptr<ngraph::Node> addPostFunction(const std::shared_ptr<ngraph::Node> &lastNode) const;
+    std::shared_ptr<ngraph::Node> addPostNodes(const ngraph::element::Type &ngPrc,
+                                               ngraph::ParameterVector &params,
+                                               const std::shared_ptr<ngraph::Node> &lastNode) const;
+protected:
+    std::shared_ptr<ngraph::Function> postFunction;
+    std::vector<postNode> postNodes;
```

Comment:
We need to determine behavior when both postFunction and PostNodes are specified. For now looks like we need to disable such case == throw exception. 
Another though here: is it possible to leave only postFunctionBuilder as parameter? Semms like we can wrap cycle over vector of postNodesBuilders as postFunctionBuilder. In that case we will have single interface.

Reply:
I decided to extract this responsibility to a different class. Now it is impossible to use postFunction and PostNodes simultaneously. In my opinion, usage of postFunction only would introduce extra complexity, since we would need to build such function from the postNodes every time we need it, and this procedure would be little bit more complicated than adding post nodes to an existing ngraph function. All mentioned changes have been done in 9a2b538.